dev/ci/docker/bionic_coq/Dockerfile

```diff
@@ -66,14 +66,15 @@ RUN opam switch create "${COMPILER}+32bit" && eval $(opam env) && \
 # EDGE switch
 ENV COMPILER_EDGE="4.14.0" \
     BASE_OPAM_EDGE="dune.2.9.3 dune-release.1.6.1 ocamlfind.1.9.1 odoc.2.0.2" \
+    CI_OPAM_EDGE="ppx_import.1.10.0 cmdliner.1.1.1 ppx_sexp_conv.v0.15.1 ppx_hash.v0.15.0 ppx_compare.v0.15.0 ppx_deriving_yojson.3.7.0 yojson.2.0.2" \
```

Comment:
Opam doesn't seem to like that yojson version

Reply:
Should we move to opam lockfiles?

Reply:
No wonder opam doesn't like being told to install two different yojson versions, I missed the one already in CI_OPAM